src/main/java/com/zaxxer/nuprocess/internal/LibC.java

```diff
@@ -27,14 +27,24 @@
 @SuppressWarnings("WeakerAccess")
 public class LibC
 {
+    
+   public static int SYS___pthread_chdir = 348; 
+    
    static {
       if (OS == OperatingSystem.MAC) {
-         O_NONBLOCK = 0x0004; // MacOS X, Freebsd
+         O_NONBLOCK = 0x0004; // MacOS X, Freebsd 
```

Comment:
Thanks for the pull request, but please remove unrelated whitespace changes.

Reply:
Whitespace has been removed, sorry about that.